docs/hub/spaces-sdks-docker-argilla.md

```diff
@@ -10,22 +10,31 @@ In the next sections, you'll learn to deploy your own Argilla app and use it for
 
 In this section, you'll learn to deploy an Argilla Space and use it for data annotation and training a sentiment classifier with [SetFit](https://github.com/huggingface/setfit/), an amazing few-shot learning library.
 
-### Duplicate the Argilla Space Template and create your Space
+### Deploy Argilla on Spaces
 
-The easiest way to get started is by [duplicating the Argilla Docker Template](https://huggingface.co/spaces/argilla/argilla-template-space?duplicate=true). You need to define the **Owner** (your personal account or an organization), a **Space name**, and the **Visibility**. To interact with the Argilla app with Python, you need to setup the visibility to `Public`. Once you are all set, click "Duplicate Space". 
+You can deploy Argilla on Spaces with just a few clicks:
+
+<a  href="https://huggingface.co/new-space?template=argilla/argilla-template-space">
+    <img src="https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg" />
+</a>
+
+You need to define the **Owner** (your personal account or an organization), a **Space name**, and the **Visibility**. To interact with the Argilla app with Python, you need to setup the visibility to `Public`.
+
+<Tip>
+If you want to customize the title, emojis, and colors of your space, go to "Files and Versions" and edit the metadata of your README.md file.
+</Tip>
 
 <div class="flex justify-center">
 <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-argilla-duplicate-space.png"/>
 </div>
 
 Once you have duplicated the space, you'll see the `Building` status and once it becomes `Running` your space is ready to go. If you don't see the Argilla login UI refresh the page.
```

Comment:
Maybe we should change the wording here as it's not a proper duplication setup

Reply:
done!